### ISSUE TITLE
Translations: Suggesion: Try weblate

### ISSUE THREAD

**Issue by Oymate** [agateau/pixelwheels#230]
Since there's a lot of translation going on, I suggest using the main weblate website to take in translations and manually merge them before major upgrades.
https://hosted.weblate.com

**Comment by agateau** [comment]
I haven't used Weblate. What would we gain from using it instead of pull requests?

**Comment by Nickoriginal** [comment]
I agree with Agateau, this is not so big project, that will require Weblate. Plus, I really don't want to register there, because AFAIK, you must do this to propose changes to the translation. Pull requests are simpler. This is my position as a translation maintainer.

**Comment by Oymate** [comment]
For translators automatic suggestion, glossary, basic grammar checking, faster contribution speed. For the maintainer, not much though.

But it generally reduces friction for translators and gives it exposure to translators. You don't need to make an account to contribute using weblate(but have an account to approve that contribution). I do agree 'proposing' changes are harder on weblate.
But if you have limited time it's best to stick to code. It's mostly if you have less than a few hour every month here it's better to stick to PR, but if you want LOTS of translators you can check weblate.
